zer0-os/zOS
# feat: add sender prefix to message preview

### What does this do?
- adds sender prefix to message preview in conversation list item.
- updates `conversation-item` tests

### Why are we making this change?
- as per figma designs. [designs](https://www.figma.com/file/aETpyuG2AKjNcQtCjldcX0/ZERO-Messenger-%2F-Web?type=design&node-id=9931-278566&mode=design&t=2ZAsx1xZv4S9lMUZ-4)

### How do I test this?
- open messenger and check prefix of last sender in the sidekick conversation item component.

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?


Before:
<img width="236" alt="Screenshot 2023-07-10 at 12 36 19" src="https://github.com/zer0-os/zOS/assets/39112648/1f97c6ba-54e3-49ce-b1cd-a7e594409a14">

After:
<img width="236" alt="Screenshot 2023-07-10 at 12 36 17" src="https://github.com/zer0-os/zOS/assets/39112648/13f8305b-a582-47fd-a942-7959bc676ca9">
